src/view/CreateQuestion.js

```diff
@@ -0,0 +1,27 @@
+
+import React from "react";
+
+const CreateQuestion = ({onCreate, onChange, newUserId, newTitle, newText, newTags}) => (
+    <div>
+        <h2>Add question</h2>
+        <div>
+            <label>User ID: </label>
+            <input value={newUserId} onChange={ e =>onChange("userId",e.target.value)}/>
```

Comment:
Alt + Shift + F is your friend. Get used to use it to format your code. Trust me, it will really help you later, especially at the beginning of your career...

Reply:
You told me the advice for formatting at the previous assignment too and I did it at the beginning of this one, but I forgot meanwhile. Thank you for the reminder!